erpnext/hr/doctype/expense_taxes_and_charges/expense_taxes_and_charges.json

```diff
@@ -56,8 +56,6 @@
   },
   {
    "columns": 2,
-   "fetch_from": "account_head.tax_rate",
-   "fetch_if_empty": 1,
```

Comment:
@rtdany10 fetch_if_empty was enabled so it won't override if the user has updated any value manually. Can you please give a more elaborate example in the description? I am unable to understand the use case for which it is causing an issue

Reply:
![Tax rate](https://user-images.githubusercontent.com/52111700/134883117-15d40205-d4f8-4a3b-9725-69553a95a87c.gif)
Sorry I didn't include it earlier.

Reply:
@rtdany10 I get the issue now. One test case seems to be failing because of this change. Can you fix that too?

Reply:
Yes, I'll fix the test case.
Also, can you give your thought on tax template against each expense?